vertx-web-openapi/src/test/java/io/vertx/ext/web/openapi/RouterBuilderIntegrationTest.java

```diff
@@ -739,6 +741,52 @@ public void testPathParameter(Vertx vertx, VertxTestContext testContext) {
     });
   }
 
+  @Test
+  public void testPathSingleParameterTemplateMatching(Vertx vertx, VertxTestContext testContext) {
+    final Checkpoint checkpoint = testContext.checkpoint(2);
+    final String expectedMessageById = "byId";
+    final String expectedMessageMine = "mine";
+
+    loadBuilderAndStartServer(vertx, VALIDATION_SPEC, testContext, routerBuilder -> {
+      defineOperation("showPetById", expectedMessageById, routerBuilder);
+      defineOperation("showMyPets", expectedMessageMine, routerBuilder);
+    }).onComplete(h -> {
+      testGetRequest200WithMessage(client, "/pets/3", expectedMessageById).send(testContext, checkpoint);
+      testGetRequest200WithMessage(client, "/pets/mine", expectedMessageMine).send(testContext, checkpoint);
+    });
+  }
```

Comment:
Related to the above comment, can you check if in the handlers mounted in this test the body is available through `routingContext.getBody()`? If not, then it means that `order` overrides the order of the BodyHandler